signer/signer.go

```diff
@@ -72,6 +73,8 @@ func NewFromFiles(keyPath, certChainPath string) (notation.Signer, error) {
 // Sign signs the artifact described by its descriptor and returns the
 // marshalled envelope.
 func (s *genericSigner) Sign(ctx context.Context, desc ocispec.Descriptor, opts notation.SignOptions) ([]byte, *signature.SignerInfo, error) {
+	logger := log.GetLogger(ctx)
+	logger.Debugf("local signing for %v", desc.Digest)
```

Comment:
The term `local signing` is confusing since it can be indeed local or plugin signing.

`generic signing` is a better term and consistent.

Reply:
Updated.